app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
```

Comment:
@jtarricone yes, or in the `export` package, or in the different package (`com.azavea.rf.batch.pkcgnamedifferentfromexportifyouwantit`) as `.airflow` looks a bit confusing, as everything in fact launched through airflow somehow. But, probably you are right, and we need to move all only 100% airflow tasks (not related to Spark) into airflow package